pydantic_ai_slim/pydantic_ai/_parts_manager.py

```diff
@@ -91,6 +92,8 @@ def handle_text_delta(
         """
         existing_text_part_and_index: tuple[TextPart, int] | None = None
 
+        # TODO: Parse out structured output or manual JSON, with a separate message?
```

Comment:
I think what you mean by this is that you want to support another form of structured output where it is read from just regular content, without using vendor API features. Is that right? If so, can you update the comment to be more clear? (I'm worried I'm misunderstanding and actually there's something that needs to be done here even for the approach implemented in this PR; regardless of which is the case, a clearer comment would help understand.)